src/site/apt/examples/set-compiler-release.apt.vm

```diff
@@ -106,3 +111,34 @@ Setting the <<<--release>>> of the Java Compiler
   [...]
 </project>
 +-----
+
+* Usage on JDK 8 - automatic detection
+
+  Since version <<<3.13.0>>> of the Compiler Plugin and default <<<javac>>> compilerId you can skip special profile.
+  The <<<--release>>> option will be automatically used according to JDK used for compilation.
```

Comment:
```suggestion
  The <<<release>>> parameter will only be effective for Java 9 or above, otherwise the <<<source>>> and <<<target>>> will be passed to the compiler.
```